blobstore-fs/src/main.rs

```diff
@@ -286,13 +332,13 @@ impl Blobstore for FsProvider {
     /// Creates a container by name, returning success if it worked
     /// Note that container names may not be globally unique - just unique within the
     /// "namespace" of the connecting actor and linkdef
-    async fn create_container(&self, ctx: &Context, arg: &ContainerId) -> RpcResult<()> {
+    async fn create_container(&self, ctx: &Context, container_id: &ContainerId) -> RpcResult<()> {
         let root = self.get_root(ctx).await?;
-        let cdir = Path::new(&root).join(arg.to_string());
+        let chunk_dir = self.resolve_subpath(&root, &container_id).await?;
 
-        info!("create dir: {:?}", cdir);
+        info!("create dir: {:?}", chunk_dir);
 
-        match std::fs::create_dir_all(cdir) {
+        match std::fs::create_dir_all(chunk_dir) {
```

Comment:
Looks like this was just a missed spot where you didn't remove the `std::fs::`.

Reply:
- [ ] remove overlooked `std::fs::` calls